### ISSUE TITLE
(NAOMI) Support for F355 (multiboard)

### ISSUE THREAD

**Issue by blisstik** [libretro/flycast#410]
Requesting for support for F355 roms below. They boot with a multiboard error message, which I hope can be faked and possibly later on be used to play versus online (wishlist).

Workaround: use Dreamcast version

```
F355
--------------------------------------------------------------------------------------------------------------
f355        M2    prior to bios f355dlx.zip,           error message (multiboard)
f355twin    M2    prior to bios f355bios.zip,          error message (multiboard)
f355twn2    M2    prior to bios f355bios.zip,          error message (multiboard)
```

**Comment by p1pkin** [comment]
arcade F355 game designed to use 2x or 4x NAOMI motherboards linked together, where 1st mobo handle game logic, I/O and sound. other mobo(s) does rendering and video output. so, even single-screen version uses 2x NAOMI motherboards per unit.
I highly doubt it can be "faked" somehow.

**Comment by blisstik** [comment]
Ahh I see. Thanks for that. I'll close for now.

**Comment by barbudreadmon** [comment]
Re-opening this, i don't see any reason to close it since afaik that's something that could be emulated.

**Comment by cinnamonmatexfce** [comment]
Are there any news / updates about emulating this game?

**Comment by Starman99x** [comment]
Any updates on this issue yet? It's been forever.

**Comment by cinnamonmatexfce** [comment]
> Any updates on this issue yet? It's been forever.

Should be fixed (at least on standalone Flycast).
Can you give it a try?

**Comment by Starman99x** [comment]
> > Any updates on this issue yet? It's been forever.
> 
> Should be fixed (at least on standalone Flycast). Can you give it a try?

I'll try the standalone, but the RetroArch core doesn't work with this yet.

**Comment by barbudreadmon** [comment]
This repo is deprecated and the libretro core is built from [upstream](https://github.com/flyinghead/flycast) (i recommend reporting issues there), so there shouldn't be differences like this between standalone and core. Are you using the official core from the libretro buildbot & online updater ? I think some 3rd-party packagers might be mistakenly still using this repo.

**Comment by Starman99x** [comment]
> This repo is deprecated and the libretro core is built from [upstream](https://github.com/flyinghead/flycast) (i recommend reporting issues there), so there shouldn't be differences like this between standalone and core. Are you using the official core from the libretro buildbot & online updater ? I think some 3rd-party packagers might be mistakenly still using this repo.

Yes I'm using the online updater core. And I recently updated all cores.

**Comment by cinnamonmatexfce** [comment]
> Any updates on this issue yet? It's been forever.

Works on the standalone (compiled) version:

![f355_triple_screen_flycast](https://github.com/libretro/flycast/assets/65762930/e292dd72-f3cd-4d46-9a1b-09cf95d35588)

**Comment by flyinghead** [comment]
The libretro core will not support naomi multiboard because it needs several flycast instances running and communicating with shared memory. This isn't compatible with the libretro API.

**Comment by Zawardo1000** [comment]
I m using standalone 2.2 android, and stuck on green screen looking for master.
There are any config setting i should do?

**Comment by DubaiShark** [comment]
The game is working with the latest Flycast v2.3 but not in Retroarch Flycast core

**Comment by zkvsky** [comment]
> The game is working with the latest Flycast v2.3 but not in Retroarch Flycast core

As mentioned above, Retroach's api is too limited, as it is now, for this to work. So go complain there if you wish so.

**Comment by DubaiShark** [comment]
> > The game is working with the latest Flycast v2.3 but not in Retroarch Flycast core
> 
> As mentioned above, Retroach's api too limited as it is now, for this to work. So go complain there if you wish so.

Hi sorry I know what was mentioned before but I got this Issue Yesterday and saw no one has updated this it may have been forgotten so I am posting my experience that's all, and it is fine for me to play them on the standalone version

Thanks
